pkg/network/files.go

```diff
@@ -9,9 +9,23 @@ import (
 
 	"github.com/containernetworking/cni/libcni"
 	"github.com/containernetworking/plugins/plugins/ipam/host-local/backend/allocator"
+	"github.com/containers/libpod/libpod"
 	"github.com/pkg/errors"
 )
 
+func GetCNIConfDir(r *libpod.Runtime) (string, error) {
```

Comment:
Rather then passing in the libpod and all of the code this pulls in, just pass in the config 
containers/common/pkg/config 
*config.Config

Reply:
I put your feedback into practice and now pass the config object.